.changeset/lazy-keys-sink.md

```diff
@@ -0,0 +1,13 @@
+---
+"fluid-framework": minor
+"@fluidframework/fluid-static": minor
+"@fluidframework/map": minor
```

Comment:
I wouldn't trust the tooling necessarily.  If adopting the changes wouldn't "just work" (i.e. a customer would have to do anything more than just update their package.json) I'd tag it as major.

Reply:
These files are consumed by tooling (change log/release notes generator) not users. I'm not sure what difference selecting major would produce in the generate release notes.

Reply:
If the tooling does something wrong then it's a bug in the tooling - the point is to annotate in alignment with semver.

Reply:
The tools are behaving as intended, and tagging stuff as major has little practical effect. Changesets is designed for semver-compliant packages. i.e. major changes should bump the major version. We don't bump the major in main. I know that right now, everything's prerelease, so we make major changes, but again, it's not the metadata that really matters. We don't use changesets for bumping versions, but if we did then tagging a change as major would cause a major version bump in that package - which is never what we want for client in main.

The best thing to do is to make it clear in the changeset contents itself what the nature of the change is. The metadata isn't that relevant.

Reply:
We should make sure that distinction is clear in the docs for using changesets, if it isn't. Prior to this discussion, I would have assumed the same as Matt.

Reply:
I'd still argue for using semver-correct tagging in the changesets - we can always decide to ignore when updating our version numbers, but without the knowledge of the author who makes the change it's difficult to look back in retrospect at a change and deduce whether it's major or minor.

Reply:
If we want semver compliant tracking, we can just use the changeset ux and bypass the wrapper.  Just run `pnpm exec changeset` instead. I can update the wiki. Less code for us to maintain that way, so we can go that route. But I think it's way more confusing to talk about major changes when we don't bump major versions. That just doesn't compute for me.

The default UX is more complex but not too bad. Biggest problem imo is that it doesn't filter out private packages so our examples get changelogs for dumb stuff, for example. Because no one actually chooses the packages carefully, they just use whatever is selected by default.

Anyway, the most important thing in a changeset is the writing that the engineer does, not the metadata.